prepare/cli.py

```diff
@@ -0,0 +1,160 @@
+import click
+
+from numpy import unique, ndarray
+from scipy.sparse.csc import csc_matrix
+from os.path import isfile, isdir, splitext, expanduser
+
+settings = dict(help_option_names=['-h', '--help'])
+
+
+@click.command()
+@click.argument('dataset', nargs=1, metavar='<dataset: file or path to data>', required=True)
+@click.option('--layout', default='umap', type=click.Choice(['umap', 'tsne', 'umap+tsne']),
+              help='layout algorithm', show_default=True)
+@click.option('--recipe', default='none', type=click.Choice(['none', 'seurat', 'zheng17']),
+              help='preprocessing to run', show_default=True)
+@click.option('--output', default='', help='save a new file to filename')
+@click.option('--set-obs-names', default='', help='named field to set as index for obs')
+@click.option('--set-var-names', default='', help='named field to set as index for var')
+@click.option('--make-obs-names-unique', default=True, is_flag=True, help='ensure obs index is unique', show_default=True)
+@click.option('--make-var-names-unique', default=True, is_flag=True, help='ensure var index is unique', show_default=True)
+@click.option('--sparse', default=False, is_flag=True, help='whether to force sparsity', show_default=True)
+@click.option('--overwriting', default=False, is_flag=True, help='allow file overwriting', show_default=True)
```

Comment:
`help=whether to allow file overwriting`  to match other flags

Reply:
fixed